src/xml_interface.h

```diff
@@ -17,7 +17,7 @@ check_for_node(pugi::xml_node node, const char *name)
 }
 
 std::string get_node_value(pugi::xml_node node, const char *name,
-  bool lowercase=false, bool strip=false);
+                           bool lowercase=true, bool strip=true);
```

Comment:
I would still prefer these guys to be false by default (i.e., no extra work by default). For any kind of numeric data we are reading from XML, there is generally no reason to lowercase and strip the string. Taking a look at where we are using `get_node_value` on the develop branch, most cases actually use the `false` default.

Reply:
Good point, most of our data is numeric.  I've switched back to `false`.